lib/register.php

```diff
@@ -221,6 +221,13 @@ function gutenberg_get_edit_post_url( $post_id ) {
  * @return string Edit post link.
  */
 function gutenberg_filter_edit_post_link( $url, $post_id, $context ) {
+	$sendback = wp_get_referer();
+	if ( $sendback && (
+		 strpos( $sendback, 'post.php' ) !== false ||
+		 strpos( $sendback, 'post-new.php' ) !== false ) ) {
```

Comment:
Should the space indenting here be a tab? It's not clear by the PHP standards, but the JavaScript standards are more specific about this case:

>When a conditional is too long to fit on one line, successive lines must be indented one extra level to distinguish them from the body.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#multi-line-statements

i.e.

```php
if ( $sendback && (
		strpos( $sendback, 'post.php' ) !== false ||
		strpos( $sendback, 'post-new.php' ) !== false ) ) {
	return $url;
}
```